src/RemoteSymbolProvider/MicrosoftSymbolServerSymbolProviderIntegrationTest.cpp

```diff
@@ -56,13 +55,15 @@ TEST(MicrosoftSymbolServerSymbolProviderIntegrationTest, RetrieveWindowsPdbAndLo
 
   orbit_qt_utils::MainThreadExecutor executor{};
 
-  const std::string valid_module_name{"d3d11.pdb"};
-  const std::string valid_module_build_id{"FF5440275BFED43A86CC2B1F287A72151"};
-  orbit_symbol_provider::ModuleIdentifier valid_module_id{valid_module_name, valid_module_build_id};
+  constexpr std::string_view kValidModuleName{"d3d11.pdb"};
+  constexpr std::string_view kValidModuleBuildId{"FF5440275BFED43A86CC2B1F287A72151"};
```

Comment:
Again, why are these `string_view`s just to be turned into `string`s?

Reply:
Done.